build/docker/updateSchema.sh

```diff
@@ -10,6 +10,16 @@ set -ex
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 cd ${DIR}
 
+# For #1366 the migration hits deadlock issues if run in parallel, so
+# to avoid this, serialize the steps using --pool-size 1
 java -jar schema/fhir-persistence-schema-*-cli.jar \
   --prop-file db2.properties --schema-name FHIRDATA --update-schema \
-  --pool-size 20
+  --pool-size 1
```

Comment:
- [ ] Team needs to get this in release notes (note to anyone doing release notes)

Reply:
posted a draft to https://github.com/IBM/FHIR/releases/edit/untagged-70be46cd90040fe8ecb5
it has no content, but this should go in there. 

I suspect, we have to maintain this going forward?

Reply:
you weren't kidding when you said no content :-)
i'll go drop the release template in there at least

Reply:
Do we need an issue created to cover this?

Reply:
I'm working on the release notes now.